system-int-test/test/cmr/system_int_test/ingest/granule_ingest_test.clj

```diff
@@ -73,6 +75,19 @@
                       (:concept-id coll1) (:concept-id coll2))]
              errors)))))
 
+;; Verify that granule indexing is not being changed too quickly.
```

Comment:
I was just thinking we don't necessarily want to see the two huge list difference. we just want to detect the change.
So add the following to let:
expected-string "okay with changing the search"
actual-string (if (= allowed-granule-index-fields actual-granule-index-fields)
"okay with changing the search"
"granule index changed, wait till next sprint to change search")]
then do (is (= expected-string actual-string)) check. sth like that. that way, people don't have to see the huge list of difference and then have to go to the test to read the doc string to find out what it is about.